Packs/ApiModules/Scripts/CrowdStrikeApiModule/CrowdStrikeApiModule.py

```diff
@@ -114,7 +114,7 @@ def _get_token(self, force_gen_new_token=False):
         """
         now = datetime.now()
         ctx = get_integration_context()
-        if not ctx or force_gen_new_token:
+        if not ctx or ctx.get('generation_time') is None or force_gen_new_token:
```

Comment:
```suggestion
        if not ctx or ctx.get('generation_time', force_gen_new_token):
```

Reply:
Do you mean to `or **not** ctx.get('generation_time', force_gen_new_token)`? if so its done